### ISSUE TITLE
I really love your work, however

### ISSUE THREAD

**Issue by elsheikh21** [yu4u/convnet-drawer#8]
I really love your work, as I said earlier but why don't you add the other layers, (Batch Normalization, Activation, Dropout, Zero Padding 2D, Max Pooling 2D), or at least package your code?

I am ready and willing to contribute with you, if you want..

**Comment by yu4u** [comment]
Thank you for your suggestion. Max Pooling 2D is implemented.
I think the other layers are somewhat annoying if they are visualized.
One solution is to add these information (e.g. activation or bn) to the text under each layer.
I had this idea before but not implemented. I'll try later.

**Comment by elsheikh21** [comment]
My bad, for mentioning something that was already implemented 😄 
Seems like you know that it easy, do it man and package. You are hindering your work this way

**Comment by Kogl1n** [comment]
Be aware that you can do BN before activation (e.g. dense, BN, softmax). And that is done regularly since it gains better results. It's kind of sad that the model summary does not incorporate that important information.

**Comment by elsheikh21** [comment]
Yes, it is. However, as I said before I can contribute to the project with
you if you want
